docs/specs/ecma2Yaml.yml

```diff
@@ -109,3 +109,117 @@ outputs:
+              <Member Id="M:CatLibrary.Cat`2.#ctor(`0)" />
+            </Type>
+          </Namespace>
+        </Framework>
+      _themes/ContentTemplate/schemas/NetNamespace.schema.json: |
+        {}
+      _themes/ContentTemplate/schemas/NetMember.schema.json: |
+        {}
+  https://github.com/ops/ecma-yaml:
+  - files:
+      cat/xml/CatLibrary/Cat`2.xml: |
+        <Type Name="Cat&lt;T,K&gt;" FullName="CatLibrary.Cat&lt;T,K&gt;">
+          <TypeSignature Language="C#" Value="public class Cat&lt;T,K&gt; where T : class, new() where K : struct" />
+          <TypeSignature Language="ILAsm" Value=".class public auto ansi serializable beforefieldinit Cat`2&lt;class .ctor T, struct .ctor (class System.ValueType) K&gt; extends System.Object" />
+          <TypeSignature Language="DocId" Value="T:CatLibrary.Cat`2" />
+          <AssemblyInfo>
+            <AssemblyName>CatLibrary</AssemblyName>
+            <AssemblyVersion>1.0.0.0</AssemblyVersion>
+          </AssemblyInfo>
+          <TypeParameters>
+            <TypeParameter Name="T">
+            </TypeParameter>
+            <TypeParameter Name="K">
+            </TypeParameter>
+          </TypeParameters>
+          <Base>
+            <BaseTypeName>System.Object</BaseTypeName>
+          </Base>
+          <Interfaces />
+          <Docs>
+            <typeparam name="T">This type should be class and can new instance.</typeparam>
+            <typeparam name="K">This type is a struct type, class type can't be used for this parameter.</typeparam>
+          </Docs>
+          <Members>
+            <Member MemberName=".ctor">
+              <MemberSignature Language="C#" Value="public Cat (T thisType);" />
+              <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(!T thisType) cil managed" />
+              <MemberSignature Language="DocId" Value="M:CatLibrary.Cat`2.#ctor(`0)" />
+              <MemberType>Constructor</MemberType>
+              <AssemblyInfo>
+                <AssemblyName>CatLibrary</AssemblyName>
+                <AssemblyVersion>1.0.0.0</AssemblyVersion>
+              </AssemblyInfo>
+              <Parameters>
+                <Parameter Name="thisType" Type="T" />
+              </Parameters>
+              <Docs>
+                <param name="thisType">This parameter type defined by class.</param>
+                <summary>
+                    Constructor with one generic parameter.
+                    </summary>
+                <remarks>To be added.</remarks>
+                <exception cref="T:None.Existing.Ex">This exception type does not exist.</exception>
+              </Docs>
+            </Member>
+          </Members>
+        </Type>
+      
+outputs:
+  cat/.errors.log: |
+    {"message_severity":"warning","code":"ECMA2Yaml_ExceptionTypeNotFound","message":"Referenced exception type not found: None.Existing.Ex", "file":"*xml/CatLibrary/Cat`2.xml*"}
```

Comment:
Pending https://dev.azure.com/ceapex/Engineering/_workitems/edit/230289 to fix fallback file path